cases/user-case-zhuanzhuan.md

```diff
@@ -0,0 +1,156 @@
+---
+title: TiDB 分布式数据库在转转公司的应用实践
+author: ['孙玄','陈东','冀浩东']
+date: 2018-05-29
+summary: “目前 TiDB 在转转线上已经接入消息、风控两套 OLTP 以及一套风控 OLAP 集群。” 
```

Comment:
Do not use double quotes in summary descriptions, especially Chinese punctuation。 @husiyu